mission-control/docs/registry/aws.md

```diff
@@ -10,6 +10,24 @@ helm repo update
 helm install {release-name} flanksource/mission-control-aws
 ```
 
+After running `helm install` you should get a success message:
+
+```sh
+NAME: {release-name}
+LAST DEPLOYED: Thu Feb 14 19:00:32 2024
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+AWS scraper added
+```
+
+When you go to the catalog now, you can now see all the AWS Resources
+
+![AWS Catalog](/img/aws-registry-catalog-scraper.png)
+
+
 ### Values
```

Comment:
nameOverride and fullnameOverride we shouldn't document, 

I don't think we should include the full connection details table here, rather add an example, and then link to schema and EnvVar reference

Reply:
For cost reporting also include details on what needs to be setup with a link to the guide,  and an example +  the IAM policy required

e.g.

```
	{
			"Effect": "Allow",
			"Action": [
				"athena:GetQueryExecution",
				"athena:GetQueryResults",
				"athena:StartQueryExecution"
			],
			"Resource": "arn:aws:athena:eu-west-1:765618022540:workgroup/primary"
		},
```